pkg/build/builder/cmd/scmauth/sslcert.go

```diff
@@ -0,0 +1,42 @@
+package scmauth
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"path/filepath"
+	"strings"
+)
+
+const (
+	SSLCertName      = "ssl-cert"
```

Comment:
if we wanted to allow client cert auth, we'd need both the cert and key, and would set them in git config thusly:

http.sslCert = ...
http.sslKey = ...